src/client/pages/VergunningDetail/BlauweZoneParticulieren.tsx

```diff
@@ -0,0 +1,34 @@
+import { defaultDateFormat } from '../../../universal/helpers';
+import InfoDetail, {
+  InfoDetailGroup,
+} from '../../components/InfoDetail/InfoDetail';
+import { BZP } from '../../../server/services/vergunningen';
+
+export function BlauweZoneParticulieren({ vergunning }: { vergunning: BZP }) {
```

Comment:
Wat mij betreft gebruik je voor de bestandsnaam het zaaktype, net als de andere vergunningen. In dit geval BZP.tsx